docs/docsite/keyword_desc.yml

```diff
@@ -7,6 +7,7 @@ always: List of tasks, in a block, that execute no matter if there is an error i
 any_errors_fatal: Force any un-handled task errors on any host to propagate to all hosts and end the play.
 async: Run a task asynchronously if the C(action) supports this; value is maximum runtime in seconds.
 become: Boolean that controls if privilege escalation is used or not on :term:`Task` execution.
+become_exe: Path to the executable used to elevate privileges.
```

Comment:
Might be a good idea to link to the become plugin page for this or above saying this is implemented by the become plugin itself. I know `runas` has no use for this value.